tsconfig.build.json

```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "./tsconfig.json",
+    "compilerOptions": {
+        "incremental": false
+    }
+}
```

Comment:
Let's just drop the `incremental` flag in the main `tsconfig.json` if it's problematic...

Reply:
`incremental` flag will only be problematic when we are trying to produce the package-ready folders, because it will include `tsconfig.buildinfo` file which is unnecessary, and the file is quite large too. Otherwise, it can act as a cache to improve compilation speed, although I believe the differences is negligible

Reply:
I see. What about excluding it from the npm bundle like so: https://github.com/InseeFrLab/keycloakify/blob/e2e0d62560dc8348e3af70bc63701564b40252a7/package.json#L44  

(I should have done it already)